sub_scripts/launcher.sh

```diff
@@ -179,7 +179,7 @@ LXC_START () {
 			sleep 1
 		done
 		echo ""
-
+		sleep 30
```

Comment:
We can't add a 30s sleep on each start of the container ! That's way too much.
Either it's conditional or you have to find another way to know that the container is ready.

Reply:
I changed it to only apply to aarch64